src/routes/mapbox.tsx

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
```

Comment:
I think it should be more of a utility function and should be put under src/state/actions?

Reply:
Ok-will change that and check lint